vmdk_plugin/utils/fs/fs.go

```diff
@@ -60,20 +60,23 @@ type VolumeDevSpec struct {
 // DevAttachWaitPrep create the watcher.
 func DevAttachWaitPrep(name string, devPath string) (*inotify.Watcher, bool) {
 	watcher, errWatcher := inotify.NewWatcher()
+	needToSkipInotify := false
```

Comment:
I think rather than introducing new variable, follow suggestions in error;
```
	if errWatcher != nil {
		log.WithFields(log.Fields{"Name": name}).Error("Failed to create watcher, skip inotify ")
		return nil, true
	} 
	err := watcher.Watch(devPath)
	if err != nil {
		log.WithFields(log.Fields{"Name": name}).Error("Failed to watch /dev, skip inotify ")
                return nil, true
	}
        err := watcher.Watch(devPath)
         if err != nil {
                   log.WithFields(log.Fields{"Name": name}).Error("Failed to watch /dev, skip inotify ")
                   return nil, true
         }
```